### PR TITLE
EDI@Energy KohlrAHBi FV2210 Changes 2024-05-21T17-03-01

### PR DESCRIPTION
**Pull request by hf-kklein**
Automated changes by [create-pull-request](https://github.com/peter-evans/create-pull-request) GitHub action

### CODE REVIEW

File: FV2210/REMADV/csv/33001.csv [hunk 1]
```diff
@@ -100,14 +100,15 @@
...
+[9] Wenn SG7 AJT+A32/ A99+E_0503 vorhanden 
+[59] Wenn SG7 AJT+A99+E_0459 vorhanden"
 94,Nähere Erläuterung des Abweichungsgrundes,SG7,FTX,4451,ABO,,Information überAbweichung,,
 95,Nähere Erläuterung des Abweichungsgrundes,SG7,FTX,4440,,,Freier Text,,
 96,Trennung von Positions- u. Summenteil,,,,,,,Muss,
 97,Trennung von Positions- u. Summenteil,,UNS,0081,S,,Trennung von Positions-und Summenteil,X,
 98,Summenbetrag,,,,,,,Muss,
 99,Summenbetrag,,MOA,5025,12,,Überweisungsbetrag,X,
-100,Summenbetrag,,MOA,5004,,,Geldbetrag,X [930] [502],[502] Hinweis: Bei positivem Betrag ergibt sich eine Forderung des
+100,Summenbetrag,,MOA,5004,,,Geldbetrag,X [930] [502],[502] Hinweis: Bei positivem
```
Comment (hf-krechan) [inline FV2210/REMADV/csv/33001.csv:111]: oh no do we lose data here?

File: FV2210/UTILMD/flatahb/11086.json
Comment (hf-krechan) [inline FV2210/UTILMD/flatahb/11086.json:1]: do we only change the guid for entries which have some changes?

Comment (hf-kklein) [reply FV2210/UTILMD/flatahb/11086.json:1]: Something Like that, have to check the Code to be sure. But i think we dont Write the file if nothing but the guids would have changed

File: FV2210/UTILMD/flatahb/11223.json
Comment (hf-krechan) [inline FV2210/UTILMD/flatahb/11223.json:1]: Did they change something in FV2210 😆 ?

File: FV2210/REMADV/csv/33001.csv (hunk 1 above)
Comment (hf-kklein) [reply FV2210/REMADV/csv/33001.csv:111]: Tests, we Need more tests;)

**Pull request merged**